pages/index.vue

```diff
@@ -45,26 +45,26 @@ const contentToUse = computed(() => {
 				<VCol cols="auto">
 					<VBtn
 						size="large"
+						role="button"
```

Comment:
don't do this. `<VBtn to="/">` renders an anchor tag `<a href="/">`, so it should be targeted with `getByRole("link")`